app/src/main/res/values-pt/strings-pt.xml

```diff
@@ -1,496 +1,521 @@
+    <string name="message_to_display_when_bgl_hits">Mensagem a mostrar quando Glicémia atinge o valor especial acima</string>
+    <string name="text_to_display_when_hitting_special_value">Texto a mostrar quando valor especial é atingido</string>
+    <string name="special_value">Valor Especial</string>
+    <string name="history">História</string>
+    <string name="statistics">Estatísticas</string>
+    <string name="snooze_alert">Snooze Alert</string>
+    <string name="less_common_settings">Definições menos comúns</string>
+    <string name="view_recent_errors_warnings">Ver Erros/Avisos recentes</string>
+    <string name="other_settings">Outras definições</string>
+    <string name="older_gratuitous_wakelocks">Outros Wakelocks que podem fazer perder muita bateria mas podem ser necessários pelo NightWatch ou Android Wear</string>
+    <string name="use_excessive_wakelocks">Utilizar Wakelocks excessivos</string>
+    <string name="speak_readings">Falar Leituras</string>
+    <string name="extra_status_line">Linha extra de estado</string>
+    <string name="dexcom_share_server_upload">Dexcom Share Server Upload</string>
+    <string name="upload_data_to_dex_servers">Enviar dados para os servidores da Dexcom para que possa usa-los nas apps Dexcom</string>
+    <string name="manage_followers">Gerir Seguidores</string>
+    <string name="manage_existing_followers">Gira os seus seguidores e convide novos.</string>
+    <string name="glucose_calibration_and_other_alerts">Glicémia, Calibração e Outros alertas</string>
+    <string name="alert_volume_profile">Perfil de Volume para Alertas</string>
+    <string name="smart_alerting">Alertas inteligentes</string>
+    <string name="dont_alarm_during_phone_calls">Não alertar durante chamadas</string>
+    <string name="start_snoozed">Começar com Snooze</string>
+    <string name="adjust_all_by_a_factor">Ajustar todos por um fator</string>
+    <string name="reset">Reiniciar</string>
+    <string name="pebble_and_android_wear_options">Opções Pebble e Android Wear</string>
+    <string name="customize_colours">Personalizar Cores</string>
+    <string name="xdrip_plus_color_settings">Definiçoes de cores xDrip+</string>
+    <string name="glucose_values_and_lines">Valor de Glicémia e Linhas</string>
+    <string name="high_glucose_values">Valores Glicémia Hiper</string>
+    <string name="in_range_glucose_values">Valores Glicémia no algo</string>
+    <string name="default_color_selected">Cor pré-definida Selecionado</string>
+    <string name="xdrip_plus_extra_settings">Definições extra xDrip+</string>
+    <string name="example_chart">Gráfico exemplo</string>
+    <string name="treatments_prediction_curves">Tratamentos / Curva preditiva</string>
+    <string name="carbs_per_unit">HC por Uni.</string>
+    <string name="everyday">Todo o dia</string>
+    <string name="insulin_sensitivity_glucose_drop_per_unit">Fator de sensibilidade à insulina: baixa de glicémia por Uni</string>
```

Comment:
It should be "Uni." which is a short for "Unidade" - do you want me to change it to "Uni." or do you prefer to use the full word everywhere?